nominal/core/client.py

```diff
@@ -602,6 +634,52 @@ def create_streaming_connection(
         )
         return Connection._from_conjure(self._clients, connection_response)
 
+    def get_template(self, rid: str) -> Template:
+        response = self._clients.template.get_template(
+            self._clients.auth_header, rid)
+        return Template._from_conjure(self._clients, response)
+
+    # TODO: fix types
+    def _create_workbook(
+            self,
+            title: str,
+            description: str,
+            is_draft: bool,
+            data_scope: scout_notebook_api.NotebookDataScope,
+            content: any,
+            layout: any,
+            charts: any
+    ) -> Workbook:
+        request = scout_notebook_api.CreateNotebookRequest(
+            title=title,
+            description=description,
+            is_draft=is_draft,
+            data_scope=data_scope,
+            stateAsJson="{}",
+            charts=charts,
+            layout=layout,
+            content=content,
+        )
+        response = self._clients.notebook.create_notebook(
+            self._clients.auth_header, request)
+        return Workbook._from_conjure(self._clients, response)
+
+    # TODO: This should probably allow for passing either a run_rid or asset_rid.
+    def create_workbook_from_template(self, run_rid: str, template_rid: str) -> Workbook:
```

Comment:
put `template_rid` in the argument list first then. that way in the future when we add support for assets here we can change the signature to 

```py
..., run_rid: str | None = None, asset_rid: str | None = None
```